cypress/e2e/API/getCustomers.cy.js

```diff
@@ -0,0 +1,80 @@
+const API_URL = Cypress.env('API_URL')
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+
+describe('EngageSphere', () => {
+  it('Should return a list of customers', () => {
+    cy.request('GET', CUSTOMERS_API_URL)
+      .then(({status, body}) => {
+        expect(status).to.eq(200);
+        expect(body).to.have.property('customers');
+        expect(body.customers).to.be.an('array');
+        expect(body.customers.length).to.be.greaterThan(0);
+      });
+  });
+
+  it('Should return an error for a negative page number', () => {
+    cy.request({
+      method: 'GET',
+      url: `${CUSTOMERS_API_URL}?page=-1`,
+      failOnStatusCode: false,
+    }).then(({status, body}) => {
+      expect(status).to.eq(400);
+      expect(body).to.have.property('error');
```

Comment:
Que tal verificar a mensagem de erro propriamente dita?

Reply:
O mesmo vale para outros testes de cenários de erro.

Reply:
A propósito, que tal testar também o cenário onde `page=0`, visto que a mensagem de erro retornada é `Invalid page or limit. Both must be positive numbers.`

Perceba o **positive numbers**.

Reply:
O mesmo vale para `limit=0`.